transform/app/tasks/utils/save_error_log.py

```diff
@@ -0,0 +1,28 @@
+import json
+import logging
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+
+def save_error_log_to_json(error_log: dict) -> None:
+    """Saves error log dictionary to JSON files, one per collection with errors."""
+    log_directory = None
```

Comment:
there is such variable already https://github.com/cyfronet-fid/eosc-search-service/blob/3149215239f70ccbe9fb2f7a0a66f15bd97da96c/transform/app/logger.py#L9

Reply:
I could use it and take the parent directory. I wanted to do it more dynamically, but simpler is better. Changed.